src/internal/operators/audit.ts

```diff
@@ -57,7 +57,7 @@ export function audit<T>(durationSelector: (value: T) => ObservableInput<any>):
     let durationSubscriber: Subscriber<any> | null = null;
     let isComplete = false;
 
-    const endDuration = () => {
+    const signalDuration = () => {
```

Comment:
Nit: I'm not sure this name change is better. This is to signal the end of the duration. "signalDuration" could be signalling the start?